meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend

```diff
@@ -48,6 +48,8 @@ SERVICE_FILE = "ofono.service"
 SERVICE_FILE_halium = "ofono-halium.service"
 
 do_install_append() {
+    # We install ril_subscription.conf with ofono-conf already, so removing it here so we don't have issues while creating the image
+    rm -rf ${D}/etc/ofono/ril_subscription.conf
```

Comment:
@tofee Not sure this is the best or cleanest way to do it. Might be good to check if the file exists before deleting it?

Reply:
let's use something like 
`[ -f ${D}/etc/ofono/ril_subscription.conf ] && rm -f ${D}/etc/ofono/ril_subscription.conf`